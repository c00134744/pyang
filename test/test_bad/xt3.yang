module xt3 {
  namespace "urn:uri:xt3";
  prefix "xt3";
  import xt4 { prefix xt4; }
  organization "hopp";
  contact "hej";
  description "arne" + /* hmm*/ ' ' + 'anka';
  revision '2005-01-01' { description "initinla"; }

  extension test {
     argument name;
  }

  typedef bazInt {
    type int16;
  }

  typedef yt {
    type string {
      pattern "[x-y]";
    }
  }

  typedef t0 {
    type xt4:yt {
      length "1..5 | 11..20";
      pattern "[x-y]*";
    }
  }

  extension hemm {
     argument data {
         yin-element true;
     }
  }

}
  

