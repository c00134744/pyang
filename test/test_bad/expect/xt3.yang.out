xt3.yang:4: [1] UNEXPECTED_KEYWORD_1
